modules/rating/src/main/java/glicko2/RatingCalculator.java

```diff
@@ -260,10 +284,11 @@ private double outcomeBasedRating(Rating player, List<Result> results) {
    *
    * @param phi
    * @param sigma
+   * @param elapsedRatingPeriods
    * @return new rating deviation
    */
-  private double calculateNewRD(double phi, double sigma) {
-    return Math.sqrt( Math.pow(phi, 2) + Math.pow(sigma, 2) );
+  private double calculateNewRD(double phi, double sigma, double elapsedRatingPeriods) {
+    return Math.sqrt( Math.pow(phi, 2) + elapsedRatingPeriods * Math.pow(sigma, 2) );
```

Comment:
so RD is capped before storing, does that mean it can sometimes be ridiculously high for a rating calculation though?

Reply:
The new RD value only affects future calculations, so capping outside should be fine